### DISCUSSION TITLE
How can I use @next/font with mantine?

### DISCUSSION THREAD

**Discussion by iamcombo** [mantinedev/mantine#3527] [Feature requests]
I'm using next@13 and I would like to use @next/font feature, but I don't know how set it up with mantine provider.

**Comment by rtivital** [comment]
Use variable – https://nextjs.org/docs/api-reference/next/font#css-variables
Assign variable to theme properties like described in typography documentation – https://mantine.dev/theming/typography/#change-fonts

**Reply by iamcombo** [reply]
cool! thank you
